components/application-deployers/org.wso2.carbon.application.deployer.synapse/pom.xml

```diff
@@ -25,6 +25,10 @@
             <groupId>org.wso2.carbon.mediation</groupId>
             <artifactId>org.wso2.carbon.mediation.initializer</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I added the deployer for synapse library in to deploymentEngine (in method getClassMediatorDeployer() ). so I need this dependency.